kubeshop/helm-charts
# fix: testkube tls error

## Pull request description 

Fix annoying TLS error spam in Operator

How to test:
* deploy testkube
* create a test
* check operator logs and verify TLS error is not present

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested on cluster

## Breaking changes

-

## Changes

- cleans up cert-manager leftovers and invalid review versions for admission and conversion

## Fixes

- Operator spamming tls error due to wrongly configured webhooks
